hack/tests/subcommand-scorecard.sh

```diff
@@ -16,36 +16,43 @@ set -ex
 pushd test/test-framework
 
 # test to see if scorecard fails when version is v1alpha2 and when external plugins are configured
-operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep '^.*error validating plugin config.*$'
+if ! $(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep -q '^.*error validating plugin config.*$'); then
+	echo "expected scorecard to fail when version is v1alpha2 and when external plugins are configured"
+	exit 1
+fi
 
 # test to see if v1alpha2 is used from the command line
-commandoutput="$(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH_V1ALPHA2" 2>&1)"
-failCount=`echo $commandoutput | grep -o "fail" | wc -l`
-expectedFailCount=3
-if [ $failCount -ne $expectedFailCount ]
-then
-	echo "expected fail count $expectedFailCount, got $failCount"
-	exit 1
+if ! commandoutput="$(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH_V1ALPHA2" 2>&1)"; then 
+	echo $commandoutput
+	failCount=`echo $commandoutput | grep -o "fail" | wc -l`
+	expectedFailCount=3
+	if [ $failCount -ne $expectedFailCount ]
+	then
+		echo "expected fail count $expectedFailCount, got $failCount"
+		exit 1
+	fi
 fi
 
 # test to see if list flag work
-commandoutput="$(operator-sdk scorecard --version v1alpha2 --list --selector=suite=basic --config "$CONFIG_PATH_V1ALPHA2" 2>&1)"
-labelCount=`echo $commandoutput | grep -o "Label" | wc -l`
-expectedLabelCount=3
-if [ $labelCount -ne $expectedLabelCount ]
-then
-	echo "expected label count $expectedLabelCount, got $labelCount"
-	exit 1
+if ! commandoutput="$(operator-sdk scorecard --version v1alpha2 --list --selector=suite=basic --config "$CONFIG_PATH_V1ALPHA2" 2>&1)"; then
```

Comment:
Should this invocation with `--list` return 0? Looks like we're expecting it to return non-zero?

Reply:
fixed in latest update.